portal-kernel/src/com/liferay/portal/kernel/workflow/WorkflowEngineManagerUtil.java

```diff
@@ -25,28 +24,32 @@
 public class WorkflowEngineManagerUtil {
 
 	public static String getKey() {
-		return _workflowEngineManager.getKey();
+		return _KEY;
 	}
 
 	public static String getName() {
-		return _workflowEngineManager.getName();
+		return _NAME;
 	}
 
 	public static Map<String, Object> getOptionalAttributes() {
-		return _workflowEngineManager.getOptionalAttributes();
+		return _optionalAttributes;
 	}
 
 	public static String getVersion() {
-		return _workflowEngineManager.getVersion();
+		return _VERSION;
 	}
 
 	public static boolean isDeployed() {
-		return _workflowEngineManager.isDeployed();
+		return true;
```

Comment:
@JanisZhang When `WorkflowEngineManagerImpl` is an OSGi service, the existence of this service represents the "deployed" state of the workflow engine module.

Please first investigate the possibility of moving the usages in portal-kernel/impl out. If there are too many dependencies chained from `WorkflowEngineManagerUtil` in portal-kernel/impl, please make sure the boolean value returned by this method does represent the state of the module.